types/yargs/index.d.ts

```diff
@@ -148,15 +148,15 @@ declare namespace yargs {
             description: string,
             builder?: BuilderCallback<T, U>,
             handler?: (args: ArgumentsCamelCase<U>) => void | Promise<void>,
-            middlewares?: MiddlewareFunction[],
+            middlewares?: Array<MiddlewareFunction<U>>,
```

Comment:
I don't really understand why these are being changed to `Array<MiddlewareFunction<U>>` and not just `MiddlewareFunction<U>[]`, but, it's not semantically different, so...

Reply:
Using `MiddlewareFunction<U>[]`, I get a tslint error for `array-type`.

```
Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead.
```

[tslint docs](https://palantir.github.io/tslint/rules/array-type/)

Reply:
Huh, TIL.

Reply:
... and I just pushed the change by mistake. 🤦‍♂️